packages/devtools_app/lib/src/app_size/app_size_screen.dart

```diff
@@ -93,6 +99,43 @@ class _AppSizeBodyState extends State<AppSizeBody>
     addAutoDisposeListener(_tabController);
   }
 
+  Future<void> maybeLoadAppSizeFiles() async {
+    final queryParams = loadQueryParams();
+    if (queryParams.containsKey(primaryAppSizeFilePropertyName)) {
+      preLoadingData = true;
+      final primaryAppSizeFile = await server.requestPrimaryAppSizeFile(
+          queryParams[primaryAppSizeFilePropertyName]);
+      DevToolsJsonFile secondaryAppSizeFile;
+      if (queryParams.containsKey(secondaryAppSizeFilePropertyName)) {
+        secondaryAppSizeFile = await server.requestSecondaryAppSizeFile(
+            queryParams[secondaryAppSizeFilePropertyName]);
+      }
+
+      if (primaryAppSizeFile != null) {
+        preLoadingData = true;
+        if (secondaryAppSizeFile != null) {
+          controller.loadDiffTreeFromJsonFiles(
+            oldFile: secondaryAppSizeFile,
+            newFile: primaryAppSizeFile,
+            onError: (error) => Notifications.of(context).push(error),
+          );
+          _tabController.animateTo(tabs.indexOf(diffTab));
+        } else {
+          controller.loadTreeFromJsonFile(
+            primaryAppSizeFile,
+            (error) => Notifications.of(context).push(error),
```

Comment:
same possible issue about possibly pushing notifications to an disposed context.